homeassistant/components/velbus/__init__.py

```diff
@@ -62,7 +62,11 @@ def callback():
         load_platform(hass, 'sensor', DOMAIN,
                       discovery_info['sensor'], config)
 
+    def syn_clock(self):
+        controller.sync_clock()
+
     controller.scan(callback)
+    hass.services.async_register(DOMAIN, 'sync_clock', syn_clock)
```

Comment:
Add a `services.yaml` file to the component folder that describes this service.  Take a look at the `hangouts` component for similar examples.